tools/topology/topology2/doc/topology2-generate-contents.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2023 Intel Corporation. All rights reserved.
+# This script generates the contents page and stores it in extra-contents/contents.doxy
+
+generate_contents ()
+{
+	echo "# This contents list is automatically generated by $1."
```

Comment:
Pretty sure you can use "$0" here and reserve arguments for if/when they're really needed.